MAUI/Popup/popup-positioning.md

```diff
@@ -143,6 +143,67 @@ private void ClickToShowPopup_Clicked(object sender, EventArgs e)
 
 ![Displaying a .NET MAUI Popup at Absolute position.](Images/popup-positioning/maui-popup-absolute-positioning.png)
 
+### Show absolute positioning in MVVM
+
+The [StartX](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_StartX) and [StartY](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_StartY) properties of the `SfPopup` control define the initial coordinates for the popup to display.

```

Comment:
The [StartX](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_StartX) and [StartY](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_StartY) properties of the `SfPopup` control are used to define the coordinates for the popup to display.

Reply:
updated as per request